apps/www/registry/default/ui/carousel.tsx

```diff
@@ -240,6 +241,7 @@ const CarouselNext = React.forwardRef<
       )}
       disabled={!canScrollNext}
       onClick={scrollNext}
+      style={{ touchAction: "manipulation", ...props.style }}
```

Comment:
@mukul-turing same here: Maybe there is a classname we can use instead of adding the CSS using the style property?

[like this](https://github.com/huntabyte/shadcn-svelte/pull/654/files#diff-76c7c20f4c1cffbe752d54b93a687df80adb73e2e62de3b5670a9f83a748abd1R22)?

Reply:
Thanks @davidjerleke for the feedback, I will update the PR. Found it here https://tailwindcss.com/docs/touch-action

Reply:
Thanks for the swift response @mukul-turing. I’ll resolve my comments 👍🏻.